src/us/kbase/workspace/database/mongo/MongoWorkspaceDB.java

```diff
@@ -331,6 +333,42 @@ private static IndexSpecification idxSpec(
 				getIndexOptions(options));
 	}
 	
+	// ew. Not sure how to make this less gross though.
```

Comment:
I can think of options for other languages but don't think they would translate to Java 😕